staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -223,3 +223,13 @@ func prefixFromUserAgent(u string) string {
 	}
 	return buf.String()
 }
+
+var createOptionsGVK = metav1.SchemeGroupVersion.WithKind("CreateOptions")
```

Comment:
global var without additional value. Would inline it.

Reply:
Thanks, I've removed all these and simplified the TypeMeta writes into more concise inline statements.